atuin/src/command/client/search/interactive.rs

```diff
@@ -225,7 +225,7 @@ impl State {
         is_down: bool,
     ) -> InputAction {
         if is_down {
-            if enable_exit && self.results_state.selected() == 0 {
+            if enable_exit && self.results_state.selected() == 0 && settings.keys.scroll_exits {
```

Comment:
```suggestion
            if settings.keys.scroll_exits && enable_exit && self.results_state.selected() == 0 {
```
It's logically equivalent, but we can check the setting before entering the implementation for `scoll_exits`. The check `enable_exit && self.results_state.selected()` is a part of the implementation of `scroll_exits`.

Reply:
yes, this makes sense.